network-area-diagram/src/main/java/com/powsybl/nad/svg/SvgWriter.java

```diff
@@ -162,6 +206,7 @@ private void draw2Wt(XMLStreamWriter writer, BranchEdge edge) throws XMLStreamEx
         if (BranchEdge.PST_EDGE.equals(edge.getType())) {
             drawPstArrow(writer, edge);
         }
+
```

Comment:
To avoid spoiling the history you'd better remove formatting changes (except if you changed code next to it).